[AutoInstall] prefer wheels whenever possible (Follow-up fix)

JonWBedard opened the pull request:
#### 219eb0bb7b43cbb8441b80ae3cb88fc55c6328d7
<pre>
[AutoInstall] prefer wheels whenever possible (Follow-up fix)
<a href="https://bugs.webkit.org/show_bug.cgi?id=263119">https://bugs.webkit.org/show_bug.cgi?id=263119</a>
<a href="https://rdar.apple.com/116915892">rdar://116915892</a>

Reviewed by Elliott Williams and Sam Sneddon.

Wheel installs of rapidfuzz aren&apos;t valid for all configurations, and
it doesn&apos;t take long to install manually.

* Tools/Scripts/libraries/webkitscmpy/setup.py: Bump version.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/__init__.py: Opt out of wheel for rapidfuzz.

Canonical link: <a href="https://commits.webkit.org/270059@main">https://commits.webkit.org/270059@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/b38cb5279e524a460902dc6a7934386fab13740c

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/24467 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/3017 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/25728 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/26594 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/22505 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/24736 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/4659 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/449 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/22914 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/24711 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/47/builds/2082 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/21133 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/27180 "Built successfully") | 
| [✅ 🧪 webkitpy](https://ews-build.webkit.org/#/builders/6/builds/24633 "Passed tests") | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/1817 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/10/builds/22058 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/28260 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/22306 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/22382 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/26053 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/44/builds/1756 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/50/builds/82 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🧪 services](https://ews-build.webkit.org/#/builders/28/builds/24235 "Passed tests") | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/46/builds/3040 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/43/builds/2209 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/3118 "Built successfully and passed tests") | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/45/builds/2131 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
<!--EWS-Status-Bubble-End-->

gsnedders commented on Tools/Scripts/libraries/webkitscmpy/webkitscmpy/__init__.py:
```diff
@@ -56,7 +56,7 @@ def _maybe_add_webkitcorepy_path():
 
 if sys.version_info > (3, 6):
-    AutoInstall.register(Package('rapidfuzz', Version(2, 11, 1), implicit_deps=['pyparsing']))
+    AutoInstall.register(Package('rapidfuzz', Version(2, 11, 1), implicit_deps=['pyparsing'], wheel=False))
```
Everywhere else we use it, we have a comment saying _why_ we're using `wheel=False`. We should at least comment this, so we know _when_ we can remove the `wheel=False` argument. It seems like https://github.com/maxbachmann/RapidFuzz/commit/abcf645ec3a4563a0d90db56bd61389bec5ed60c should practically fix the universal macOS wheels, which seems like the problem.

gsnedders replied:
To copy/paste from elsewhere:

`rapidfuzz-2.11.1-cp38-cp38-macosx_10_9_universal2.whl` is broken, `rapidfuzz-2.11.1-cp38-cp38-macosx_10_9_x86_64.whl` seems fine.

Specifically, with the universal2 wheel, we get:

```
ImportError: dlopen(/Volumes/Data/worker/Style-EWS/build/Tools/Scripts/libraries/autoinstalled/python-3-x86_64/rapidfuzz/distance/metrics_cpp.cpython-38-darwin.so, 0x0002): symbol not found in flat namespace (__ZN4Avx215OsaDistanceInitEP14_RF_ScorerFuncPK10_RF_KwargsxPK10_RF_String)
```

JonWBedard committed:
[AutoInstall] prefer wheels whenever possible (Follow-up fix)
https://bugs.webkit.org/show_bug.cgi?id=263119
rdar://116915892

Reviewed by Elliott Williams and Sam Sneddon.

Wheel installs of rapidfuzz aren't valid for all configurations, and
it doesn't take long to install manually.

* Tools/Scripts/libraries/webkitscmpy/setup.py: Bump version.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/__init__.py: Opt out of wheel for rapidfuzz.

Canonical link: https://commits.webkit.org/270059@main
